lib/theming/src/convert.ts

```diff
@@ -1,7 +1,6 @@
 import { opacify } from 'polished';
-
 import { background, typography, color } from './base';
-import { Theme, Color, ThemeVars } from './types';
+import { StorybookTheme, Color, ThemeVars } from './types';
```

Comment:
note that within the `theming` package you have to reference `StorybookTheme` and not `Theme` because `Theme` doesn't exist yet - it's only added after bundling and into the reexporting file